scalableminds/webknossos
# N:M datastores (backend-code)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up two instances of webknossos and use standalone datastores (https://github.com/scalableminds/webknossos/wiki/Set-up-a-standalone-datastore-locally)
- add an foreign DataStore and DataSet with e.g. `http://localhost:9000/api/datasets/addForeign?url=http://localhost:9191&dataStoreName=standalone-9191&dataSetName=ROI2017_wkw2`
- you should see the new dataset in the dataset gallery and you should be able to view the dataSet
- use pgAdmin to delete the DataSet again (not the DataStore)
- do  `http://localhost:9000/api/datasets/addForeign?url=http://localhost:9191&dataStoreName=standalone-9191&dataSetName=ROI2017_wkw2` again
- even though the dataStore already exists, this should still work

### Issues:
- fixes #2945

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
